docs/web3-eth-admin.rst

```diff
@@ -0,0 +1,423 @@
+-------
+
+
+``Promise<ArrayBuffer>`` - List of all connected peers.
+
+
+-------
+Example
+-------
+
+
+.. code-block:: javascript
+
+    web3.eth.admin.getPeers().then(console.log);
+    > [{
+            caps: ["eth/61", "eth/62", "eth/63"],
+            id: "08a6b39263470c78d3e4f58e3c997cd2e7af623afce64656cfc56480babcea7a9138f3d09d7b9879344c2d2e457679e3655d4b56eaff5fd4fd7f147bdb045124",
+            name: "Geth/v1.5.0-unstable/linux/go1.5.1",
+            network: {
+                localAddress: "192.168.0.104:51068",
+                remoteAddress: "71.62.31.72:30303"
+            },
+            protocols: {
+                eth: {
+                    difficulty: 17334052235346465000,
+                    head: "5794b768dae6c6ee5366e6ca7662bdff2882576e09609bf778633e470e0e7852",
+                    version: 63
+                }
+            }
+        }, /* ... */ {
+            caps: ["eth/61", "eth/62", "eth/63"],
+            id: "fcad9f6d3faf89a0908a11ddae9d4be3a1039108263b06c96171eb3b0f3ba85a7095a03bb65198c35a04829032d198759edfca9b63a8b69dc47a205d94fce7cc",
+            name: "Geth/v1.3.5-506c9277/linux/go1.4.2",
+            network: {
+                localAddress: "192.168.0.104:55968",
+                remoteAddress: "121.196.232.205:30303"
+            },
+            protocols: {
+            eth: {
+                difficulty: 17335165914080772000,
+                head: "5794b768dae6c6ee5366e6ca7662bdff2882576e09609bf778633e470e0e7852",
+                version: 63
+            }
+        }
+    }]
+
+------------------------------------------------------------------------------
+
+
+setSolc
+=====================
+
+.. code-block:: javascript
+
+    web3.eth.admin.setSolc(string, [, callback])
+
+Sets the Solidity compiler path to be used by the node when invoking the eth_compileSolidity RPC method
+
+----------
+Parameters
+----------
+
+
+1. ``String`` - The path of the solidity compiler.
+2. ``Function`` - (optional) Optional callback, returns an error object as first parameter and the result as second.
+
+
+-------
+Returns
+-------
+
+
+``Promise<string>`` - A message.
+
+
+-------
+Example
+-------
+
+
+.. code-block:: javascript
+
+    web3.eth.admin.setSolc("/usr/bin/solc").then(console.log);
+    > "solc, the solidity compiler commandline interface\nVersion: 0.3.2-0/Release-Linux/g++/Interpreter\n\npath: /usr/bin/solc"
+
+------------------------------------------------------------------------------
+
+
+startRPC
+=====================
+
+.. code-block:: javascript
+
+    web3.eth.admin.startRPC(host, port, cors, apis [, callback])
+
+It starts an HTTP based JSON RPC API webserver to handle client requests. All the parameters are optional.
+
+----------
+Parameters
+----------
+
+1. ``host`` - ``String`` - (optional) The network interface to open the listener socket on (defaults to "localhost").
+2. ``post`` - ``hex | number`` - (optional) The network port to open the listener socket on (defaults to 8545).
```

Comment:
`string | number` as `hex` is misleading and people can see that as `string | number` so it reads `string | number | number`

Reply:
cool.